pkg/operator/controller/garden/care/health.go

```diff
@@ -79,21 +80,26 @@ var (
 		virtualGardenPrefix+v1beta1constants.ETCDEvents,
 	)
 
+	requiredMonitoringDeployments = sets.New(
+		v1beta1constants.DeploymentNameKubeStateMetrics,
+		v1beta1constants.DeploymentNamePlutono,
+	)
+
 	virtualGardenMonitoringSelector = labels.SelectorFromSet(map[string]string{v1beta1constants.LabelRole: v1beta1constants.LabelMonitoring})
 )
 
-// GardenHealth contains information needed to execute health checks for garden.
-type GardenHealth struct {
+// Health contains information needed to execute health checks for garden.
+type Health struct {
```

Comment:
```suggestion
type health struct {
```
We don't have to export it. All its properties are private, so you have to create instances using `NewHealth` anyway.